docs/schema/reference.md

```diff
@@ -274,7 +274,7 @@ The contract section is used to provide details of contracts that have been ente
 
 ### Implementation
 
-Implementation information can be updated over the course of a contract. It belongs nested within the contract it relates to. Implementation blocks include the following elements:
+Implementation information can be updated over the course of a contract. The `implementation` array belongs nested within the contract it relates to. The `Implementation` subschema includes the following fields:
```

Comment:
`contracts.implementation` is an object.

```suggestion
Implementation information can be updated over the course of a contract. The `implementation` object belongs nested within the contract it relates to. It includes the following fields:
```

Reply:
oops, forgot the array was nested within it!